parsing/parser.mly

```diff
@@ -1133,10 +1134,10 @@ parse_pattern:
 functor_arg:
     (* An anonymous and untyped argument. *)
     x = mkrhs(LPAREN RPAREN {"*"})
-      { x, None }
+      { x, None, $startpos }
```

Comment:
I would pass `$startpos` as the first value of the tuple, to respect a sort of spatial intuition.